components/dialog/index.css

```diff
@@ -81,16 +89,6 @@
 	overflow: hidden;
 }
 
-.spectrum-Dialog--sizeS {
-	--spectrum-standard-dialog-max-width: var(--spectrum-standard-dialog-maximum-width-small);
-	inline-size: var(--mod-standard-dialog-max-width-small, var(--spectrum-standard-dialog-max-width));
-}
-
-.spectrum-Dialog--sizeL {
-	--spectrum-standard-dialog-max-width: var(--spectrum-standard-dialog-maximum-width-large);
-	inline-size: var(--mod-standard-dialog-max-width-large, var(--spectrum-standard-dialog-max-width));
-}
-
```

Comment:
To better fit how we've been writing our CSS, I moved the t-shirt variables up with the rest of our variables, and then left the inline-size property on the `.spectrum-Dialog` selector. Any reservations about this move?